src/stores/results.ts

```diff
@@ -10,34 +9,59 @@ export const useResultsStore = defineStore("results", {
       return this.resultats?.droitsEligibles || []
     },
     benefitTreeGroupExperiment(): (StandardBenefit | BenefitGroup)[] {
-      const bafaGroup: BenefitGroup = {
-        benefits: [],
-        id: "bafa-bafd-group",
-        label: "Aides BAFA et BAFD",
-        logoPath: "/img/benefits/logo-bafa-bafd.png",
-        description:
-          "Différents organismes peuvent vous aider à financer votre formation BAFA ou BAFD.",
-        redirectionPage: "bafa-bafd",
-      }
+      const groups: BenefitGroup[] = [
+        {
+          benefits: [],
+          id: "bafa-bafd-group",
```

Comment:
```suggestion
          id: "bafa-bafd",
```

ou en tous cas, dans les URL de redirection ne pas avoir `groupe/bafa-bafd-group` mais bien `groupe/bafa-bafd`.